src/components/BestItemsSection.jsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from "react";
+import { getItems } from "../utils/api";
+import ItemsContainer from "./ItemsContainer";
+import styles from "./ItemsSection.module.css";
+
+const LIST_TYPE = "best";
+
+const BestItemsSection = ({ pageSize }) => {
+  const [bestItemList, setBestItemList] = useState([]);
+
+  const loadBestItemList = async (options) => {
+    const result = await getItems(options);
+    if (!result) return;
+    const { list } = result;
+    setBestItemList(list);
+  };
+
+  useEffect(() => {
+    if (!pageSize) return;
+    (async () => {
+      await loadBestItemList({
+        offset: 1,
+        pageSize: pageSize,
+        orderBy: "favorite",
+        keyword: "",
+      });
+    })();
+  }, [pageSize]);
```

Comment:
loadBestItemList() 함수가 이미 비동기 함수인데 useEffect 구문안에서 또 비동기 처리를 하기 위해 즉시 실행 함수를 사용하셨군요! 별다른 의도가 있는게 아니라면, 그러실 필요 없습니다.

아래와 같은 코드 스타일이 훨씬 일반적입니다.

```suggestion
  useEffect(() => {
    if (!pageSize) return;
      loadBestItemList({
        offset: 1,
        pageSize: pageSize,
        orderBy: "favorite",
        keyword: "",
      });
  }, [pageSize]);
```